madanalysis/core/main.py

```diff
@@ -463,6 +463,20 @@ def CheckConfig2(self,debug=False):
             return False
         self.AutoSetGraphicalRenderer()
 
+        try:
+            import datetime
+            # check for updates every afternoon between 1 PM and 3 PM
+            now = datetime.datetime.now()
+            if 13 < now.hour < 15:
```

Comment:
I don't understand why exactly at this moment. Any reasons?

Reply:
No reason, I just didn't want it to annoy the user all the time, so I scheduled it.